.github/workflows/circle_artifacts.yml

```diff
@@ -3,7 +3,8 @@ name: Redirect circleci artifacts
 on: [status]
 
 permissions:
-   contents: read  # to fetch code (actions/checkout)
+  contents: read  # to fetch code (actions/checkout)
```

Comment:
I don't think you need this one actually. And https://github.com/scientific-python/circleci-artifacts-redirector-action/issues/70 is merged and `v1.1.0` is out so I'd suggest to update to that and merge this and see if it works. If it doesn't, then we could even try using https://github.com/scientific-python/circleci-artifacts-redirector-action/pull/66 in `main` for a short while and merge that PR if it does fix things...